raiden-dapp/src/registerServiceWorker.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+/* istanbul ignore file */
+
+import { register } from 'register-service-worker';
+
+if (
+  process.env.NODE_ENV === 'production' &&
+  !process.env.VUE_APP_RAIDEN_PACKAGE
+) {
+  register(`${process.env.BASE_URL}service-worker.js`, {
+    ready() {
+      console.log(
+        'App is being served from cache by a service worker.\n' +
+          'For more details, visit https://goo.gl/AFskqB'
+      );
+    },
+    registered(registration: ServiceWorkerRegistration) {
+      // Check periodically for updates every minute
+      setInterval(() => {
+        registration.update();
+      }, 1000 * 60);
```

Comment:
This will check every minute while the dapp is running whether there's a new service worker available. I first had this at `1000`ms but that would be pretty bad e.g. if a user is running a transfer and we release an update, he would be blocked from continuing and forced to refresh the dapp.